### ISSUE TITLE
Coloured syntax highlighting

### ISSUE THREAD

**Issue by lonyelon** [ostracod/breadtext#2]
Maybe you could add more colors to the syntax, for example, instead of using **KEYWORDS**, you could write the RGB value and then the keywords that will be highlighted with that color. As my english is bad, let me show you an example:
```
COMMENT
#0F0
//
#F00
and or && || if while for class
#00F
int double float string char long short
```
In this case, comments would be green, variable types blue and other things (like logical operators), red. If you approve this changes, I may work in them but only if I have free time.

**Comment by ostracod** [comment]
I think adding more keyword colors is a good idea, but I'd rather specify the keywords in categories rather than explicit colors. For example:

```
COMMENT
//
KEYWORD
and or && || if while for class
TYPE
int double float string char long short
```

Then the user could specify colors in their `.breadtextrc` file like so:

```
commentColor red
keywordColor cyan
typeColor blue
```

I'd like to only use the original 8 ANSI terminal colors. This will make BreadText compatible with more terminals.

I will probably add color customization into a future release.

**Comment by lonyelon** [comment]
True, I hadn't thought about it.

**Comment by ostracod** [comment]
Largely addressed in v1.4.0 release.
